packages/core/src/provider/base.ts

```diff
@@ -266,6 +266,15 @@ export class SmartAccountProvider<
       overrides.maxPriorityFeePerGas = maxPriorityFeePerGas;
     }
 
+    return {
+      batch,
+      overrides,
+    };
+  };
+
+  sendTransactions = async (requests: RpcTransactionRequest[]) => {
```

Comment:
@denniswon I can't remember did you add `overrides` to this as well? yea right?

Reply:
yeah I don't think the branch merged in, but i'd rebase (or whoever gets in first :))